recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
I think this is outside of the spec (http://conda.pydata.org/docs/building/pkg-name-conv.html):

```
package name
The name of a package, without any reference to a particular version. Conda package names are normalized, and may contain only lowercase alpha characters, numeric digits, underscores, or hyphens. In usage documentation, these will be referred to by package_name.
```